```
CMake Error at D:/installed/x64-windows/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake:58 (message):
  vcpkg_cmake_config_fixup was passed extra arguments:
  TARGET_PATH;share/robin-map
Call Stack (most recent call first):
  ports/robin-map/portfile.cmake:14 (vcpkg_cmake_config_fixup)
  scripts/ports.cmake:142 (include)
```

--- a/ports/robin-map/CONTROL
+++ b/ports/robin-map/CONTROL
@@ -1,3 +1,0 @@
-Source: robin-map
-Version: 0.6.3
-Description: A C++ implementation of a fast hash map and hash set using robin hood hashing

--- a/ports/robin-map/portfile.cmake
+++ b/ports/robin-map/portfile.cmake
@@ -1,15 +1,18 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO Tessil/robin-map
-    REF 622443f40544fb6a693402e69c1328d685eac939 # v0.6.3
-    SHA512 ec91ee3ce1a992cfc1b82c17d8d0d1d1af7078df9871b585526a81e1e05162b844958d3bf61badad191902d837832e29dc254755965410beac702ddf15c55a49
+    REF v0.6.3
+    SHA512 485557f300d33bda62bb8accdf246819ee8ffe956bc022e7ddca54ff6ad1a9fdb8db8d80690add3ef238e834d1eb8e2905920cb0a0674e7df010f6946d01297b
     HEAD_REF master
 )
 
-file(COPY ${SOURCE_PATH}/include/tsl DESTINATION ${CURRENT_PACKAGES_DIR}/include)
-
-file(INSTALL
-    ${SOURCE_PATH}/LICENSE
-    DESTINATION ${CURRENT_PACKAGES_DIR}/share/robin-map
-    RENAME copyright
+vcpkg_cmake_configure(
+    SOURCE_PATH ${SOURCE_PATH}
 )
+
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(CONFIG_PATH share/cmake/tsl-robin-map)
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
+
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

--- a/ports/robin-map/vcpkg.json
+++ b/ports/robin-map/vcpkg.json
@@ -1,0 +1,17 @@
+{
+  "name": "robin-map",
+  "version-semver": "0.6.3",
+  "port-version": 1,
+  "description": "A C++ implementation of a fast hash map and hash set using robin hood hashing",
+  "homepage": "https://github.com/Tessil/robin-map",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5374,7 +5374,7 @@
     },
     "robin-map": {
       "baseline": "0.6.3",
-      "port-version": 0
+      "port-version": 1
     },
     "rocksdb": {
       "baseline": "6.14.6",

--- a/versions/r-/robin-map.json
+++ b/versions/r-/robin-map.json
@@ -1,8 +1,13 @@
 {
   "versions": [
     {
-      "git-tree": "84f1433234bb4813feee71e4042174ec9e8d5a7a",
-      "version-string": "0.6.3",
+      "git-tree": "95d49e16c7a8ef523608ef8e7d8fb019085789b8",
+      "version-semver": "0.6.3",
+      "port-version": 1
+    },
+    {
+      "git-tree": "3a6eb0ea067edf2d06baa7706281caf96df4216a",
+      "version-semver": "0.6.3",
       "port-version": 0
     },
     {

